ui/apps/dev-server-ui/src/coreapi.ts

```diff
@@ -603,3 +603,92 @@ export const COUNT_WORKER_CONNECTIONS = gql`
     }
   }
 `;
+
+export const GET_EVENTS = gql`
+  query GetEventsV2(
+    $cursor: String
+    $startTime: Time!
+    $endTime: Time
+    $celQuery: String = null
+    $eventNames: [String!] = null
+    $includeInternalEvents: Boolean = true
+  ) {
+    eventsV2(
+      first: 30
+      after: $cursor
+      filter: {
+        from: $startTime
+        until: $endTime
+        query: $celQuery
+        eventNames: $eventNames
+        includeInternalEvents: $includeInternalEvents
+      }
+    ) {
+      edges {
+        node {
+          name
+          id
+          receivedAt
+          runs {
+            status
+            id
+            startedAt
+            endedAt
+            function {
+              name
+              slug
+            }
+          }
+        }
+      }
+      totalCount
```

Comment:
FYI @amh4r and I discussed changing this behavior in https://linear.app/inngest/issue/CON-25/fix-pagination-behavior-with-totalcount-on-events-page, but we can fix both cloud and dev-server together in that issue